src/decode.h

```diff
@@ -1135,23 +1155,10 @@ static inline bool PacketIsNotTunnel(const Packet *p)
  */
 static inline bool VerdictTunnelPacket(Packet *p)
 {
-    bool verdict = true;
+    bool verdict;
     SCSpinlock *lock = p->root ? &p->root->persistent.tunnel_lock : &p->persistent.tunnel_lock;
     SCSpinLock(lock);
-    const uint16_t outstanding = TUNNEL_PKT_TPR(p) - TUNNEL_PKT_RTV(p);
-    SCLogDebug("tunnel: outstanding %u", outstanding);
-
-    /* if there are packets outstanding, we won't verdict this one */
-    if (PacketIsTunnelRoot(p) && !PacketTunnelIsVerdicted(p) && !outstanding) {
-        // verdict
-        SCLogDebug("root %p: verdict", p);
-    } else if (PacketIsTunnelChild(p) && outstanding == 1 && p->root &&
-               PacketTunnelIsVerdicted(p->root)) {
-        // verdict
-        SCLogDebug("tunnel %p: verdict", p);
-    } else {
-        verdict = false;
-    }
+    verdict = VerdictTunnelPacketInternal(p);
```

Comment:
declare variable here?

Reply:
I guess I don't like to declare a var inside a lock, then use it outside. It's a matter of style I think, I don't think it has practical effect

Reply:
ok, the comment was based on the experience from the previous PRs.